vkconfig/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@
 cmake_minimum_required(VERSION 3.4)
 project(vkconfig)
 
+if (NOT DEFINED CMAKE_INSTALL_BINDIR)
+  include(GNUInstallDirs)
+endif()
+
 set(CMAKE_CXX_STANDARD 11)
```

Comment:
Maybe this part isn't needed as I am guarenteed to define CMAKE_INSTALL_PREFIX and CMAKE_INSTALL_BINDIR from the commandline.